examples/1-hello-world/index.js

```diff
@@ -0,0 +1,21 @@
+import { createServer } from "http";
```

Comment:
Useing the [`node:` protocol](https://nodejs.org/api/esm.html#node-imports) is best practice

```suggestion
import { createServer } from "node:http";
```

It makes it clear that a module is not coming from npm

Reply:
Done